cmd/dex/config.go

```diff
@@ -29,6 +29,7 @@ type Config struct {
 	OAuth2     OAuth2      `json:"oauth2"`
 	GRPC       GRPC        `json:"grpc"`
 	Expiry     Expiry      `json:"expiry"`
+	Logger     Logger      `json:"logger"`
```

Comment:
Maybe we could call this section "debug"? That way we can at least add other fields here later. I feel like we're getting a lot of config sub-sections. What do you think?